javascript/javascript3/week2/geolocation.js

```diff
@@ -0,0 +1,31 @@
+const setTimeoutPromise = (timeout) =>{
+     let promise = new Promise((resolve, reject) => {
+       let seconds = timeout * 1000;
+       setTimeout(() => {
+         resolve("Called after " + timeout + " seconds");
+       }, seconds);
+     })
+       .then((result) => console.log(result))
+       .catch((error) => console.log("error"));
+
+}
+//setTimeoutPromise(3); 
+
+
+
+
+
+  async function getCurrentLocation(postion) {
+    // const latitude = postion.coords.latitude;
+    // const longitude = postion.coords.longitude;
+    try {
+      fetch(
+        `https://www.openstreetmap.org/#map=18/${latitude}/${longitude}`
```

Comment:
I see you are not setting any values for latitude and longitude
You can see how it works here:
https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API